includes/class-wc-gateway-ppec-settings.php

```diff
@@ -303,8 +353,14 @@ public function currency_has_decimal_restriction() {
 	 */
 	public function get_paypal_locale() {
 		$locale = get_locale();
+
 		if ( ! in_array( $locale, $this->_supported_locales ) ) {
-			$locale = 'en_US';
+			// Mapping some WP locales to PayPal locales
+			if( array_key_exists( $locale, $this->_locales_mapping ) ) {
```

Comment:
There's a missing space after the `if` here. 

eg `if ( array_key_exists( $locale, $this->_locales_mapping ) ) {`

I noticed you're also using `array_key_exists()` rather than `isset()`. They both will do the same thing here, however `isset()` is a little faster. 

I don't really mind in this case since the performance gain is minuscule and it's not running billions of times. But just wanted to note it.